test/evergreen.yml

```diff
@@ -5043,6 +5043,88 @@ tasks:
+    tags: ["long-perf"]
+    patch_only: true
+    depends_on:
+      - name: perf-test-long-500m-btree-populate
+    commands:
+      - command: timeout.update
+        params:
+          exec_timeout_secs: 10800
+          timeout_secs: 10800
+      # Fetch the compile artifacts.
+      - func: "fetch artifacts"
+      # Fetch the database created by perf-test-long-500m-btree-populate task. This will be used
+      # as the source for live restore.
+      - func: "fetch artifacts"
+        vars:
+          dependent_task: perf-test-long-500m-btree-populate
+          destination: "wiredtiger/cmake_build/bench/wtperf/SOURCE"
+      # Create an empty home directory to be used as the destination for live restore.
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/cmake_build/bench/wtperf"
+          shell: bash
+          script: |
+            set -o errexit
+            set -o verbose
+            mkdir WT_TEST_0_0
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: 500m-btree-50r50u-live-restore.wtperf
+          maxruns: 1
+          no_create: true
+          wtarg: -args ['"-C create,statistics=(fast),statistics_log=(json,wait=1,sources=[file:]),live_restore=(enabled=true,path=SOURCE)"'] -ops ['"read", "update", "warnings", "max_latency_read_update"']
+      - func: "upload stats to atlas"
+        vars:
+          test-name: 500m-btree-50r50u-live-restore.wtperf
+      - func: "upload stats to evergreen"
+        vars:
+          test-name: 500m-btree-50r50u-live-restore.wtperf
+      - func: "cleanup"
+      
+  - name: perf-test-live-restore-no-server
+    tags: ["long-perf"]
+    patch_only: true
+    depends_on:
+      - name: perf-test-long-500m-btree-populate
+    commands:
+      - command: timeout.update
+        params:
+          exec_timeout_secs: 10800
+          timeout_secs: 10800
+      # Fetch the compile artifacts.
+      - func: "fetch artifacts"
+      # Fetch the database created by perf-test-long-500m-btree-populate task. This will be used
+      # as the source for live restore.
+      - func: "fetch artifacts"
+        vars:
+          dependent_task: perf-test-long-500m-btree-populate
+          destination: "wiredtiger/cmake_build/bench/wtperf/SOURCE"
+      # Create an empty home directory to be used as the destination for live restore.
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/cmake_build/bench/wtperf"
+          shell: bash
+          script: |
+            set -o errexit
+            set -o verbose
+            mkdir WT_TEST_0_0
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: 500m-btree-50r50u-live-restore-no-server.wtperf
+          maxruns: 1
+          no_create: true
+          wtarg: -args ['"-C create,statistics=(fast),statistics_log=(json,wait=1,sources=[file:]),live_restore=(enabled=true,path=SOURCE,threads_max=0)"'] -ops ['"read", "update", "warnings", "max_latency_read_update"']
```

Comment:
Why not put the live restore config inside the `conn_config` param in the .wtperf file?

Reply:
Good idea

Reply:
On second thoughts, the config takes a path to `source` which is something that is environment dependent and in this case, is created above in the `fetch artifacts` function. I think it makes most sense to leave it here so anyone reading this task can see we're creating that folder and then using in the configuration.

Reply:
Although it would make sense given the file names

Reply:
@ajmorton what do you think?

Reply:
I'm ok with moving it into the .wtperf file. I don't think we'll ever pass a `source` directory other than SOURCE here. I'm also not sure how the two different strings interact. so having just the one location should make our lives easier.

Reply:
Moved https://github.com/wiredtiger/wiredtiger/pull/11544/commits/51fd3e0ab67a11b32573c2d4eccd0c4ca22b760a

and created [WT-14094 Refactor evergreen perf tasks](https://jira.mongodb.org/browse/WT-14094) for more refactoring.